src/Reanimate.hs

```diff
@@ -63,7 +63,7 @@ module Reanimate
     freezeAtPercentage,
     addStatic,
     signalA,
-    -- ** Signals
+    -- ** Ease
```

Comment:
Could you change this to 'Easing functions'?

Reply:
Sure, done in next commit.